TESTS/mbed_hal/common_tickers_freq/main.cpp

```diff
@@ -30,12 +30,16 @@
 #include "hal/lp_ticker_api.h"
 #include "hal/mbed_lp_ticker_wrapper.h"
 
+#if defined(SKIP_TIME_DRIFT_TESTS)
+#error [NOT_SUPPORTED] test not supported
+#endif
+
 #if !DEVICE_USTICKER
 #error [NOT_SUPPORTED] test not supported
 #endif
 
 //FastModels not support time drifting test
-#if defined(__ARM_FM)
+#if defined(TARGET_ARM_FM)
```

Comment:
This can be removed as you got `#if defined(SKIP_TIME_DRIFT_TESTS)` above

Reply:
fixed.